app/templates/main/503.html

```diff
@@ -1,14 +1,34 @@
 {% extends "base.html" %}
 
-{% block pageTitle %}Sorry, the service is unavailable – {{config['SERVICE_NAME']}} – GOV.UK{% endblock %}
+{% block pageTitle %}Sorry, the service is unavailable{% endblock %}
```

Comment:
According to [https://design-system.service.gov.uk/patterns/service-unavailable-pages/](https://design-system.service.gov.uk/patterns/service-unavailable-pages/).
The page title should be ‘Sorry, the service is unavailable – service name – GOV.UK’

Reply:
Fixed, thanks